test/stream/stats.js

```diff
@@ -5,92 +5,47 @@ var stats = require('../../stream/stats'),
 module.exports.tests = {};
```

Comment:
It could be moved to an external module, I will open a ticket for this and the other tasks to externalise functionality not in the osm domain.